### ISSUE TITLE
x86-64 doesn't require 3DNow as the README claims

### ISSUE THREAD

**Issue by fuel-pcbox** [iximeow/yaxpeax-x86#36]
No Intel CPU has ever supported 3DNow instructions, and 3DNow isn't required by the x86-64 specifications, but the README claims it is for some reason.

**Comment by iximeow** [comment]
yeah, that's outright wrong. i'm not sure why i wrote that. 3dnow is accepted by the decoders by default though, even though those instructions presumably #UD on real hardware from the last decade and a half. thank you for filing the issue.

incidentally that means all the helpers to configure decoders for specific microarchitectures are also wrong (since they build on `InstDecoder::minimal()` and that itself is wrong). so, i'll fix that up at some point this week..

**Comment by iximeow** [comment]
one very, very long journey later, this was a motivator to [add flags for more ISA extensions](https://github.com/iximeow/yaxpeax-x86/commit/08eed360fea81ab9328fd0859b813ee01937b5b1) (and be able to reject 3DNow instructions at all), then [try to ground feature flags in reality](https://github.com/iximeow/chip_directory/tree/no-gods-no-/x86), [catalogue the CPU families described by InstLatx64 CPUID dumps](https://github.com/iximeow/chip_directory/blob/no-gods-no-/x86/product_info.sql#L1), (finally) start [collecting docs for x86 chipsets and peripherals](https://github.com/iximeow/chip_directory/blob/no-gods-no-/x86/docs.sql#L57), and most recently [delete a bunch of redundant code between the different modes](https://github.com/iximeow/yaxpeax-x86/commit/54538a07bf4e0b8b2335e146198cda02a49589c8).

anyway. that incorrect statement is now gone from the readme and per-uarch decoders reject 3DNow instructions like they ought to, when they should. K8 and K10 support 3DNow still, the CPUID bit dropped off in Bobcat and Bulldozer. so that's one last bit (pun) to follow up on here. but that's much more likely to actually happen ""this week""

thanks for the issue.
